src/ovp_pipeline/truth_api.py

```
@@ -3442,6 +3479,22 @@ def _attach_action_queue_state(
+            for item in items
+            for path in item.get("note_paths", [])
+            if isinstance(path, str) and path.strip()
+        )
+    )
+    try:
+        capture_summaries = (
+            _collect_note_capture_summaries(vault_dir, capture_note_paths)
+            if capture_note_paths
+            else {}
+        )
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)